gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -132,7 +132,12 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel.getBottomSheetState().observe(this, this::onBottomSheetStateChange);
     viewModel.getOverlappingFeatures().observe(this, this::showFeatureSelector);
     viewModel.getOpenDrawerRequests().as(autoDisposable(this)).subscribe(__ -> openDrawer());
-    viewModel.getAddFeatureResults().as(autoDisposable(this)).subscribe(this::onFeatureAdded);
+    viewModel
+        .getAddFeatureResults()
+        .subscribeOn(schedulers.io())
```

Comment:
As a general practice, we've been adding `subscribeOn` as far upstream as possible, ie as close to the IO or other async operation as possible to make it clearer why we're subscribing on IO thread. Would you be able to move the `subscribeOn` into the stream's definition in  `HomeScreenViewModel`?

Reply:
Done